integration_tests/go.mod

```diff
@@ -121,6 +174,7 @@ require (
 
 replace (
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
-	github.com/pingcap/tidb => github.com/lhy1024/tidb v1.1.0-beta.0.20251215110526-59e2c3c2eab3
+	github.com/pingcap/tidb => github.com/lcwangchao/tidb v1.1.0-beta.0.20251229025756-9746aadede81
```

Comment:
So, client-go integration test is depending on fixed tidb commit, instead of latest?

Reply:
This dependency is only used for unistore. But we should rely on a personal branch to make sure this integration can be built because the master TiDB does not fit the API change.